jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -484,6 +484,11 @@ public String join(String prefix, String suffix, String delimiter, String[] elem
 		return String.join(prefix, suffix, delimiter, elements, size);
 	}
 	/*[ENDIF] OPENJDK_METHODHANDLES*/
+	
+	public Class<?> findBootstrapClassOrNull(String name) {
+		return VMAccess.findClassOrNull(name, ClassLoader.bootstrapClassLoader);
+	}
```

Comment:
The old signature should (eventually?) be removed:
```
public Class<?> findBootstrapClassOrNull(ClassLoader loader, String name)
```
as it has been removed from `JavaLangAccess`.

Reply:
Removed it for JDK17.